resources/js/Components/Form/ItemCreateForm.tsx

```diff
@@ -70,13 +68,19 @@ const ItemCreateForm = ({
                   tier: editData?.tier || 1,
               }
 
+    const itemStepAdditionalHint =
+        type === 'Create'
+            ? ' Please note that the first item is the one that gets attached to a character should you choose to accept the Magic item as a reward.'
+            : ''
+    const itemStepHint = {
+        text: `Fill out the following fields with your Magic Item details.${itemStepAdditionalHint}`,
```

Comment:
confused about this, why not just make one singular variable for this, if the else condition returns `''`, there is no need of ternary operator

Reply:
I only want part of the hint displayed in the edit mode.

Reply:
yeah, you can do that in a single variable

Reply:
`text: `Fill out the following fields with your Magic Item details.${type === 'Create' && itemStepAdditionalHint}`

Reply:
probably not worth declared as a constant at that point also, the rule is it should be declared as a constant on component level if it's re-used